pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go

```diff
@@ -509,6 +512,25 @@ func (c *completedConfig) withUserAPIServer(delegateAPIServer genericapiserver.D
 	return server.GenericAPIServer, &legacyStorageVersionMutator{version: userapiv1.SchemeGroupVersion, storage: storage}, nil
 }
 
+func (c *completedConfig) withAggregator(delegatedAPIServer genericapiserver.DelegationTarget) (*genericapiserver.GenericAPIServer, error) {
```

Comment:
does this mean we have a full kube-aggregator in the chain? Does it serve APIService objects?

Reply:
Wait, why we need aggregator in the openshift API? We are already running one, in the k8s API, you should not create additional one.